modernizer-maven-plugin/src/main/resources/modernizer.xml

```diff
@@ -1244,15 +1272,29 @@ violation names use the same format that javap emits.
   <violation>
     <name>java/io/FileOutputStream."&lt;init&gt;":(Ljava/lang/String;)V</name>
     <version>7</version>
+    <until>11</until>
     <comment>Prefer java.nio.file.Files.newOutputStream(java.nio.file.Paths.get(String))</comment>
   </violation>
 
+  <violation>
+    <name>java/io/FileOutputStream."&lt;init&gt;":(Ljava/lang/String;)V</name>
+    <version>11</version>
+    <comment>Prefer java.nio.file.Files.newOutputStream(java.nio.file.Path.of(String))</comment>
```

Comment:
Could you change the parameters to not include the full path but lease the method name with the full path?  #312 discusses this.

Reply:
IIUC with "full path" you actually mean the package name?